extensions/infinispan-embedded/runtime/src/main/java/io/quarkus/infinispan/embedded/runtime/graal/FixJMXClasses.java

```diff
@@ -0,0 +1,52 @@
+package io.quarkus.infinispan.embedded.runtime.graal;
+
+import java.util.Properties;
+
+import org.infinispan.configuration.global.GlobalJmxStatisticsConfiguration;
+import org.infinispan.configuration.parsing.ConfigurationBuilderHolder;
+import org.infinispan.configuration.parsing.Parser;
+import org.infinispan.configuration.parsing.XMLExtendedStreamReader;
+import org.infinispan.factories.GlobalComponentRegistry;
+
+import com.oracle.svm.core.annotate.Alias;
+import com.oracle.svm.core.annotate.Substitute;
+import com.oracle.svm.core.annotate.TargetClass;
+
+public class FixJMXClasses {
+}
+
+@TargetClass(GlobalComponentRegistry.class)
+final class SubstituteGlobalComponentRegistry {
+    @Substitute
+    private boolean isMBeanServerRunning() {
+        return false;
+    }
+
+    @Substitute
+    protected synchronized void addShutdownHook() {
+        // Don't do anything or do we want to?
```

Comment:
probably best to log a warning.

Reply:
Sure.

Reply:
Looking closer, just doing nothing should be acceptable.